app/vmui/packages/vmui/src/utils/time.ts

```diff
@@ -227,3 +227,22 @@ export const getTimezoneList = (search = "") => {
 export const setTimezone = (timezone: string) => {
   dayjs.tz.setDefault(timezone);
 };
+
+const isValidTimezone = (timezone: string) => {
+  try {
+    dayjs().tz(timezone);
+    return true;
+  } catch (e) {
+    return false;
+  }
+};
+
+export const getBrowserTimezone = () => {
+  const timezone = dayjs.tz.guess();
+  const isValid = isValidTimezone(timezone);
+  return  {
+    isInvalid: !isValid,
```

Comment:
I think it is better to make the field consistent and do not do reverse logic. I suggest to use `IsValid` field in the object and use it in other places

Reply:
updated